toolkit/scripts/toolchain/container/toolchain_build_in_chroot.sh

```diff
@@ -828,15 +828,15 @@ popd
 rm -rf diffutils-3.8
 touch /logs/status_diffutils_complete
 
-echo Gawk-5.1.0
-tar xf gawk-5.1.0.tar.xz
-pushd gawk-5.1.0
+echo Gawk-5.1.1
```

Comment:
@anphel31, do we update the raw toolchain within one version of Mariner? I don't remember what's our policy here now, but I don't recall other toolchain changes updating this file. But maybe they were for packages, which were not part of the raw toolchain.

Reply:
It's always a good idea to update the raw toolchain packages, but we have not been enforcing this in general in 2.0. Whether it's required depends on the nature of the CVE. If the CVE relates to a build-time issue with the tool, and a vulnerable version of the tool in the raw toolchain is used to build some other component, then the raw toolchain package should be updated.